tests/test_inside_docker.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh -xe
+
+OS_VERSION=$1
+BUILD_ENV=$2
+
+# Pre-create vdttest user so that we can create the BLAHP status cache
+# debugging logs
+useradd -m vdttest
+
+# Source repo version
+git clone https://github.com/opensciencegrid/osg-test.git
+pushd osg-test
+git rev-parse HEAD
+make install
+popd
+git clone https://github.com/opensciencegrid/osg-ca-generator.git
+pushd osg-ca-generator
+git rev-parse HEAD
+make install
+popd
+
+# Bind on the right interface and skip hostname checks.
+cat << EOF > /etc/condor/config.d/99-local.conf
+BIND_ALL_INTERFACES = true
+GSI_SKIP_HOST_CHECK=true
+ALL_DEBUG=\$(ALL_DEBUG) D_FULLDEBUG D_CAT
+SCHEDD_INTERVAL=1
+SCHEDD_MIN_INTERVAL=1
+JOB_ROUTER_POLLING_PERIOD=1
+GRIDMANAGER_JOB_PROBE_INTERVAL=1
+EOF
```

Comment:
Fine how you are doing it, but note if you want to do a here-doc without variable expansion, you can just quote the delimiter word (rather than escaping the `$` signs), eg:

```suggestion
cat << "EOF" > /etc/condor/config.d/99-local.conf
BIND_ALL_INTERFACES = true
GSI_SKIP_HOST_CHECK=true
ALL_DEBUG=$(ALL_DEBUG) D_FULLDEBUG D_CAT
SCHEDD_INTERVAL=1
SCHEDD_MIN_INTERVAL=1
JOB_ROUTER_POLLING_PERIOD=1
GRIDMANAGER_JOB_PROBE_INTERVAL=1
EOF
```

Reply:
Neato!